src_py/requirements.txt

```diff
@@ -1,17 +1,78 @@
-aniso8601==9.0.1
-certifi==2022.5.18.1
-charset-normalizer==2.0.12
-click==8.1.3
-Flask==2.1.2
-Flask-RESTful==0.3.9
-idna==3.3
-importlib-metadata==4.11.4
-itsdangerous==2.1.2
-Jinja2==3.1.2
-MarkupSafe==2.1.1
-pytz==2022.1
-requests==2.27.1
-six==1.16.0
-urllib3==1.26.9
-Werkzeug==2.1.2
-zipp==3.8.0
+apturl==0.5.2
+bcrypt==3.1.7
+blinker==1.4
+Brlapi==0.7.0
+certifi==2019.11.28
+chardet==3.0.4
+Click==7.0
+colorama==0.4.3
+command-not-found==0.3
+configobj==5.0.6
+cryptography==2.8
+cupshelpers==1.0
+dbus-python==1.2.16
+defer==1.0.6
+distlib==0.3.4
+distro==1.4.0
+distro-info===0.23ubuntu1
+duplicity==0.8.12.0
+entrypoints==0.3
+fasteners==0.14.1
+filelock==3.7.0
+future==0.18.2
+httplib2==0.14.0
+idna==2.8
+keyring==18.0.1
+language-selector==0.1
+launchpadlib==1.10.13
+lazr.restfulclient==0.14.2
+lazr.uri==1.0.3
+lockfile==0.12.2
+louis==3.12.0
+macaroonbakery==1.3.1
+Mako==1.1.0
+MarkupSafe==1.1.0
+mock==3.0.5
+monotonic==1.5
+netifaces==0.10.4
+oauthlib==3.1.0
+olefile==0.46
+paramiko==2.6.0
+pbr==5.4.5
+pexpect==4.6.0
+Pillow==7.0.0
+platformdirs==2.5.2
+protobuf==3.6.1
+psutil==5.5.1
+pycairo==1.16.2
+pycups==1.9.73
+PyGObject==3.36.0
+PyJWT==1.7.1
+pymacaroons==0.13.0
+PyNaCl==1.3.0
+PyQt5==5.14.1
+pyRFC3339==1.1
+python-apt==2.0.0+ubuntu0.20.4.5
```

Comment:
I think this is a wrong requirements.txt 
Why do we need packages that relates with ubuntu? 
On raspberry or orange pi there is debian not ubuntu and this requirements.txt will fail 

Please generate requirement.txt only with modules that you've imported in apiServer 
and Jupyter-lab 
We need the minimal requirement for it to work.